packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -291,17 +332,21 @@ export class Nodegroup extends Resource implements INodegroup {
       tags: props.tags,
     });
 
+    if (props.capacityType) {
+      resource.addPropertyOverride('CapacityType', props.capacityType.valueOf());
```

Comment:
Lets maybe already create an issue for this so we don't forget? i'd really want to get rid of this once we can.

Reply:
Yes. Noted https://github.com/aws/aws-cdk/issues/12154